Update ploidy (2/2)

sangeetashukla opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose/implementation Section
Add markdown file as a result of recoding formula per [issue 318](https://github.com/PediatricOpenTargets/ticket-tracker/issues/318)

#### What scientific question is your analysis addressing?



#### What was your approach?
Recode

`chrom == "chrY" & germline_sex_estimate == "Female" & copy.num > 0 ~ "unknown"`
to

`chrom == "chrY" & germline_sex_estimate == "Female" & copy.num > 0 ~ NA`


#### What GitHub issue does your pull request address?
[Issue 318](https://github.com/PediatricOpenTargets/ticket-tracker/issues/318)


### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?



#### Is there anything that you want to discuss further?



#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?



### Results

#### What types of results are included (e.g., table, figure)?



#### What is your summary of the results?



#### Reproducibility Checklist

<!-- Check all those that apply or remove this section if it is not applicable.-->

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

<!-- Please review https://github.com/AlexsLemonade/OpenPBTA-analysis#documenting-your-analysis -->

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

<!-- IF YOUR PULL REQUEST IS A DATA RELEASE, PLEASE REMOVE THE [HTML COMMENT TAG](https://html.com/tags/comment-tag/) FROM THE SECTION BELOW AND COMPLETE THE CHECKLIST-->

<!-- 
### Data Release Checklist
- [ ] Is the table in doc/data-file-descriptions.md up to date?
- [ ] Is doc/data-format.md up to date?
- [ ] Is doc/release-notes.md up to date?
- [ ] Is download-data.sh up to date?
- [ ] Was download-data.sh tested and did it complete without error?
-->

jharenza commented:
@sangeetashukla can you stagger this on top of #181 please merge that branch into this one, change the base branch here, and rerun?

jharenza commented:
@sangeetashukla your base branch here should be `[sangeetashukla:focal_cn_merge_run_files](https://github.com/sangeetashukla/OpenPedCan-analysis/tree/focal_cn_merge_run_files)` not dev. I will change it up above.

sangeetashukla commented:
> @sangeetashukla your base branch here should be `[sangeetashukla:focal_cn_merge_run_files](https://github.com/sangeetashukla/OpenPedCan-analysis/tree/focal_cn_merge_run_files)` not dev. I will change it up above.

From our slack discussion, I can confirm that the #181 (originally branched out from #182) is merged back into #182

runjin326 commented on analyses/focal-cn-file-preparation/README.md:
````diff
@@ -23,6 +23,7 @@ Running the following from the root directory of the repository will run the ste
 ```
 RUN_ORIGINAL=1 bash analyses/focal-cn-file-preparation/run-prepare-cn.sh
 ```
+**Temporary Note**: The `run-prepare-cn.sh` script currently downloads the `gencode.v38.annotation.gtf.gz` file since it is not available in the data directory. When it is added to the set of files within the data directory, this script will need to be updated to remove the download requirement and instead read it along with the rest of the files.
````
This will be available in data-release. We might not need to add this. Every time we run `download_data.sh` - we will get the gencode 38.

sangeetashukla replied:
@runjin326 I agree. Once the files for v11 are all ready, I can remove the download lines from this script and open another PR.

runjin326 commented on analyses/focal-cn-file-preparation/run-prepare-cn.sh:
```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
 cd "$script_directory" || exit
 
+
+GENCODE="ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz"
```
if we want to add this - maybe set it to download to `data` directory if it does not exist and the read from the data directory. So that we do not need to modify this script again.

sangeetashukla replied:
@runjin326 I think it would make more sense to completely remove the download lines. I suggest we leave this the way it is for now, and open another PR once the v11 set is ready.

runjin326 commented on analyses/focal-cn-file-preparation/run-prepare-cn.sh:
```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
+--outdir ${results_dir}
+
+
+# filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn" "cnvkit_annotated_cn_wxs" "controlfreec_annotated_cn_wxs")
```
Should we also remove these commented-out lines since it would not be needed at this moment?

sangeetashukla replied:
@runjin326 Sorry I missed removing them. May be I can remove them in the next PR while also removing the download lines for gencode v38?

runjin326 reviewed (changes requested):
See my individual comments - other than those, everything else looks fine and this would be ready for merge once my comments are addressed.

runjin326 commented on analyses/focal-cn-file-preparation/run-prepare-cn.sh:
```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
 histologies_file=${data_dir}/histologies.tsv
-gtf_file=${data_dir}/gencode.v27.primary_assembly.annotation.gtf.gz
+#gtf_file=${data_dir}/gencode.v38.primary_assembly.annotation.gtf.gz
+gtf_file=gencode.v38.annotation.gtf.gz
```
oh - also, did not catch it before, but we want the primary_assembly gencode.

sangeetashukla replied:
@runjin326 Yes, thanks for bringing this up. I did notice this is not the primary_assembly file when I downloaded it, but I was unable to find a different link for v38 This was the only link I found for v38 annotation. Would you mind looking it up and sharing it with me please?

sangeetashukla commented:
> See my individual comments - other than those, everything else looks fine and this would be ready for merge once my comments are addressed.

@runjin326 I have addressed all your comments, please review again when you get a chance.

jharenza commented:
I am going to close this so we can start fresh
